app/src/main/java/org/p2p/wallet/kyc/StrigaFragmentFactory.kt

```diff
@@ -0,0 +1,43 @@
+package org.p2p.wallet.kyc
+
+import androidx.fragment.app.Fragment
+import org.p2p.wallet.kyc.model.StrigaKycStatusBanner
+import org.p2p.wallet.smsinput.SmsInputFactory
+import org.p2p.wallet.striga.finish.StrigaSignupFinishFragment
+import org.p2p.wallet.striga.onboarding.StrigaOnboardingFragment
+import org.p2p.wallet.striga.signup.ui.StrigaSignUpFirstStepFragment
+import org.p2p.wallet.striga.user.model.StrigaUserStatusDestination
+
+class StrigaFragmentFactory() {
+
+    fun kycFragment(status: StrigaKycStatusBanner): Fragment {
```

Comment:
As I know what you need is just return `StrigaKycFragment()` (our wrapper) for any status, it seems sumsub handles all this stuff by itself
@gslevinkov please, elaborate on this

Reply:
yep, Striga fetches status by themself and show needed screens